### PR TITLE
Update page with jQuery AJAX

### PR DESCRIPTION
**Pull request by dodecaphonix**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -2,10 +2,21 @@
...
   <body>
 
-      <h1>So it begins...</h1>
-      {{ state }}
+    <h1>So it begins...</h1>
+
+    <div id="pile0"></div>
+    <div id="pile1"></div>
+    <div id="pile2"></div>
+    <div id="pile3"></div>
```
Comment (romainrichard) [inline templates/index.html]: Why add more than 3 piles now?

File: static/game_operations.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+function newGame() {
+  $.getJSON( "/new", function( json ) {
+    $( "#pile0" ).html( json.state[0] );
+    $( "#pile1" ).html( json.state[1] );
+    $( "#pile2" ).html( json.state[2] );
+    $( "#pile3" ).html( json.state[3] );
+    $( "#pile4" ).html( json.state[4] );
+    console.log( "JSON Data: " + json.state );
```
Comment (romainrichard) [inline static/game_operations.js]: Lots of duplicated code. No way to factorize?

File: templates/index.html (hunk 1 above)
Comment (dodecaphonix) [reply templates/index.html]: To illustrate that they won't be populated to dave.

File: static/game_operations.js (hunk 2 above)
Comment (dodecaphonix) [reply static/game_operations.js]: May be able to `for loop` it... I wanted to keep it very simple for now.

Comment (romainrichard) [reply static/game_operations.js]: Not just that. The whole function is duplicated.

Comment (dodecaphonix) [reply static/game_operations.js]: I see. Yeah, there must be a better way. I'll fix it.

**Pull request merged**